app/controllers/public_controller.rb

```diff
@@ -104,7 +104,11 @@ def contact_us
 
   def commission_histories
     if params[:validator_id]
-      @validator = Validator.find(params[:validator_id])
+      begin
+        @validator = Validator.find(params[:validator_id])
+      rescue ActiveRecord::RecordNotFound
```

Comment:
We have to rescue the error ? I think that when the  RecordNotFound occurs Rails goes to render_404 by default.

Reply:
I'm rescuing the error so that appsignal does not report unnecessary information